<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTestWndClass</class>
 <widget class="QWidget" name="CTestWndClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CTestWnd</string>
  </property>
  <widget class="QPushButton" name="btn_topleft">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>左上</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_bottomright">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>右下</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_topright">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>右上</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_bottomleft">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>左下</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="infobox_text">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ctestwnd.qrc"/>
 </resources>
 <connections/>
</ui>
